src/config.ts

```diff
@@ -270,6 +270,16 @@ export const defaultConfig: Config = {
     whisker: {}
   },
 
+
+  callout: {
+    calloutAngle: 45,
+    calloutOffset: 0,
+    calloutLength: 30,
+    labelOffset: 5,
```

Comment:
5 is a bit too much IMHO.  I think 2 is better.  In any case, your documentation says default is `0`, which doesn't match the number here.